include/open62541/util.h

```diff
@@ -175,6 +175,83 @@ UA_EXPORT extern const UA_ReferenceTypeAttributes UA_ReferenceTypeAttributes_def
 UA_EXPORT extern const UA_DataTypeAttributes UA_DataTypeAttributes_default;
 UA_EXPORT extern const UA_ViewAttributes UA_ViewAttributes_default;
 
+/**
+ * Role Bitfield Representation
+ * ----------------------------
+ *
+ * The build macro ``UA_ROLESET_MAX`` defines how many roles a server can handle
+ * at most. The size of the role-set bitmaps depends on this. */
+
+#define UA_ROLEINDEX_ANONYMOUS 0
+#define UA_ROLEINDEX_AUTHENTICATEDUSER 1
+#define UA_ROLEINDEX_OBSERVER 2
+#define UA_ROLEINDEX_OPERATOR 3
+#define UA_ROLEINDEX_ENGINEER 4
+#define UA_ROLEINDEX_SUPERVISOR 5
+#define UA_ROLEINDEX_CONFIGUREADMIN 6
+#define UA_ROLEINDEX_SECURITYADMIN 7
+
+typedef struct {
+    UA_UInt16 bits[UA_ROLESET_MAX / 16];
```

Comment:
Using "(UA_ROLESET_MAX + 15) / 16" here might relax the current restriction of UA_ROLESET_MAX being a multiple of 16.